app/assets/javascripts/profile/email.coffee

```diff
@@ -26,6 +26,7 @@ class Email
     ev.preventDefault()
     $.ajax({type: "PUT", url: @url('resend_confirmation')})
       .success( (resp) =>
+        ev.target.disabled = true
         OX.Alert.display(message: resp.message, type: 'success', parentEl: @$el)
```

Comment:
I think it's a good idea to disable the button, but it might be better to:

* Disable button as soon as click occurs, before the ajax call.
* Re-enable the button only if the call fails, in the `.error` callback

This would prevent double clicks on the button, which I suspect is the reason for the DB error.  Robots shouldn't have access to the page, since it requires a login.